src/app/main.cpp

```diff
@@ -19,11 +19,15 @@
 ***********************************************************************************************************************/
 
 #include "../core/compositesnapindetailsdialog.h"
+#include "../core/logger/consolelogger.h"
+#include "../core/logger/filelogger.h"
```

Comment:
We should probably group loggers into a single header named "logger.h" or something.